test/shared/helpers/view.test.js

```diff
@@ -34,7 +34,9 @@ describe('view', function () {
         return {
           options: { entryPath: '/path' },
           modelUtils: {
-            underscorize: function (name) { return name; }
+            underscorize: function (name) { return name; },
+            isModel: function(model) { return model instanceof ModelClass },
```

Comment:
@saponifi3d unfortunately that won't work in this test, as ModelClass and CollectionClass are stubbed.